tests/Agent/IntegrationTests/SharedApplications/Common/MultiFunctionApplicationHelpers/MultiFunctionApplicationHelpers.csproj

```diff
@@ -157,8 +130,6 @@
   </ItemGroup>
 
   <ItemGroup> <!-- Versions below 4.5 did not support netstandard 2.0 -->
-    <PackageReference Include="NLog" Version="4.5.10" Condition="'$(TargetFramework)' == 'netcoreapp2.1'" />
-    <PackageReference Include="NLog" Version="4.5.9" Condition="'$(TargetFramework)' == 'netcoreapp2.2'" />
     <PackageReference Include="NLog" Version="4.6.8" Condition="'$(TargetFramework)' == 'netcoreapp3.1'" />
     <PackageReference Include="NLog" Version="5.0.0" Condition="'$(TargetFramework)' == 'net5.0'" />
     <PackageReference Include="NLog" Version="4.7.15" Condition="'$(TargetFramework)' == 'net6.0'" />
```

Comment:
Why does .NET 6 test an older version of NLog than .net 5? (I know you didn't introduce this, but since we're looking at this can you fix it up?